OneSignalSDK/unittest/src/test/java/com/onesignal/ShadowOSUtils.java

```diff
@@ -88,6 +89,11 @@ public static void resetStatics() {
 
    }
 
+   @Implementation
+   public static boolean isThreadRunningOnMainThread() {
```

Comment:
We shouldn't be overriding this for the tests. We will want to know in the tests if we have an issue with a network call being made on the main thread.

Reply:
yeah U r right